src/client/components/pages/entities/publisher.js

```diff
@@ -101,7 +101,7 @@ function PublisherDisplayPage({entity, identifierTypes, user}) {
 			<EntityAnnotation entity={entity}/>
 			{!entity.deleted &&
 			<React.Fragment>
-				<EditionTable editions={entity.editions} entity={entity}/>
+				<EditionTable editions={entity.editions} entity={entity} showAuthorCreditsColumn={entity.showAuthorCreditsColumn}/>
```

Comment:
I don't think we need to set showAuthorCreditsColumn in the entity at all on the server side (which means we can remove [this](https://github.com/metabrainz/bookbrainz-site/pull/957/files#diff-6e372470c7eb4abe59f52ad00d5de4cb951ea90de60d72fd8b16fdf9e8d2b852R211)); instead we can simply use this boolean prop directly right here.
```suggestion
				<EditionTable editions={entity.editions} entity={entity} showAuthorCreditsColumn/>
```

Reply:
You're right! Thanks for the suggestion :)